src/main/java/uk/gov/ons/ctp/response/action/service/impl/ActionRuleServiceImpl.java

```diff
@@ -69,10 +70,10 @@ public ActionRule updateActionRule(final ActionRule actionRule) {
       existingActionRule.setDescription(newDescription);
     }
 
-    final Integer newDaysOffset = actionRule.getDaysOffset();
-    if (newDaysOffset != null) {
+    final OffsetDateTime triggerDateTime = actionRule.getTriggerDateTime();
+    if (triggerDateTime != null) {
       needsUpdate = true;
```

Comment:
Not in scope of this change but the whole needsUpdate boolean logic seems a premature optimisation

Reply:
Might be a little out of scope, we just changed the getTriggerDateTime thing from the line above